tools/bionano/bionano_scaffold_edited.xml

```diff
@@ -0,0 +1,524 @@
+<tool id="bionano_scaffold" name="Bionano Hybrid Scaffold" version="@TOOL_VERSION@+@GALAXY_TOOL_VERSION@" profile="20.01">
+    <description>automates the scaffolding process</description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="edam_ontology"/>
+    <expand macro="requirements"/>
+    <version_command>perl /HybridScaffold/hybridScaffold.pl --version</version_command>
+    <command detect_errors="exit_code"><![CDATA[
+        #set RefAligner = '/usr/local/bin/RefAligner'
+        ## softlinks do not work
+        cp '${ngs_fasta}' ./ngs.fasta
+        && cp '${bionano_cmap}' ./bionano.cmap
+        #if $configuration_options.configuration == 'vgp'
+            && cp '${vgp_mode}' ./config.xml
+        #else
+            #set cores = os.environ.get('GALAXY_SLOTS', 2)
+            #set memory = os.environ.get('GALAXY_MEMORY_MB', 4000) / 1000
+            && cp '${configuration_file}' ./config.xml
+            && sed -i 's|attr="maxmem" val0=.* display|attr="maxmem" val0="$memory" display|' ./config.xml
+            && sed -i 's|attr="maxthreads" val0=.* display|attr="maxthreads" val0="$cores" display|' ./config.xml
+            && sed -i 's|attr="maxvirtmem" val0=.*/>|attr="maxvirtmem" val0="$cores"/>|' ./config.xml
+            && cat ./config.xml
```

Comment:
this cat should happen in both cases.... and then we can have tests that check that the core settings are correct

Reply:
The problem is that the VGP configuration file cannot be used with the input files used in the tests because the restriction enzymes are different.